test/podFailover/README.md

```diff
@@ -17,50 +17,31 @@ export REGISTRY=<your-personal-docker-repo>
 make pod-failover-test-containers
 ```
 
-### Create metrics service
-
-- Create the metrics service where the workload pod can publish it's metrics using the command mentioned below:
-
-```console
-wget https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/main_v2/test/podFailover/metrics-svc.yaml
-```
-
-- Edit the value of metrics-service-image and run:
-
-```console
-kubectl apply -f metrics-svc.yaml
-```
-
-- Create the service monitor to enable Prometheus to scrape the pod failover service metrics.
+### Run failover test
 
-```console
-wget https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/main_v2/test/podFailover/service-monitor.yaml
-kubectl apply -f service-monitor.yaml
+- Deployment via helm chart (controller/metrics)
+``` console
+helm install pod-failover-controller pod-failover-controller-chart/ --values pod-failover-controller-chart/values.yaml
```

Comment:
This command specifies the default values file, so the `--values` parameter seems redundant.